### ISSUE TITLE
Use rejected in conjunction with keep?

### ISSUE THREAD

**Issue by jValdron** [danialfarid/ng-file-upload#820]
At the moment, when the `ngf-keep` option is set to `true`, the logic seems to be to ignore updating the `ng-model-rejected`.

It seems like when you upload a single rejected file, it hits this condition: https://github.com/danialfarid/ng-file-upload/blob/05be0d93c42eb4dd00a021263e68d7c4c42e45e4/dist/ng-file-upload.js#L415 and there's no files, so it just returns.

Is there a reason why the `ngModelRejected` model doesn't get updated if there's no files? Shouldn't the `ngModelRejected` get updated before checking for the condition for files?

Here's my view:

```html
	<div class="drop-box"
		 ngf-drop
		 ngf-select
		 ng-model="files"
		 ng-model-rejected="rejectedFiles"
		 ngf-multiple="true"
		 ngf-allow-dir="true"
		 ngf-keep="true"
		 ngf-keep-distinct="true"
		 ngf-max-size="1073741824">
		Drop files here
		<small>or click to upload</small>
	</div>
```

**Comment by danialfarid** [comment]
Fixed in version 5.0.7

**Comment by jValdron** [comment]
Awesome, thanks for the quick fix! Just tried 5.0.7 and everything seems to work as expected.
